docs/src/test-cli-js.md

```diff
@@ -86,6 +86,8 @@ Complete set of Playwright Test options is available in the [configuration file]
 
 - `--browser`: Run test in a specific browser. Available options are  `"chromium"`, `"firefox"`, `"webkit"` or `"all"` to run tests in all three browsers at the same time.
 
+- `--debug`: Run tests in debug mode. Shortcut for timeout=0, maxFailures=1, PWDEBUG=1, headed=true, workers=1
```

Comment:
```suggestion
- `--debug`: Run tests in debug mode. Shortcut for `PWDEBUG=1 npx playwright test --timeout=0 --maxFailures=1 --headed --workers=1`.
```

Reply:
Let's also replace "Run in debug mode with Playwright Inspector" snippet at https://playwright.dev/docs/next/intro#command-line and https://playwright.dev/docs/next/test-cli#examples.

We can also add "run with --debug" first bullet here: https://playwright.dev/docs/next/inspector#open-playwright-inspector.